### DISCUSSION TITLE
View MusicXML File Tags

### DISCUSSION THREAD

**Discussion by bassman71** [musescore/MuseScore#17295] [Q&A] [answered]
Hello, trying to view the contents i.e. tags/values etc, of a MusicXML file, however it only renders as notation in Muse Score and text editors render as unreadable characters. Any help is much appreciated.

Rich O

**Comment by cbjeukendrup** [comment, marked as answer]
If it is a `.mxl` file, then it is a "compressed musicxml file". That means that it is actually a ZIP file, with the real MusicXML code inside it. You can view the XML code by renaming the file by replacing `.mxl` with `.zip`, and then using any ZIP program (for example, Windows Explorer's built-in tools) to unzip it. The result should be a folder containing a file that contains the MusicXML code.

**Reply by bassman71** [reply]
Thank you very much for the information.

**Comment by Jojo-Schmitz** [comment]
Only if is is a `.xml` or `.musicxml` file, it is directly readable with a text editor.
And instead of renaming the `.mxl` to `.zip`, you may be able to right-click it and use 'Open with' to use WinZip or 7Zip ("7-Zip File Manager") on it (I believe the Windows Explorer builtin tool doesn't allow for this though).
